pkg/backend/httpstate/diffs_pre_go120.go

```diff
@@ -0,0 +1,18 @@
+//go:build !go1.20
```

Comment:
We can delete this file until we have the split pre_120, post_120 setup.

Reply:
In fact, I'm pretty sure this will fail to compile on Go 1.19 since there will be two definitions of unsafeAsString: one in the untagged file and one in this file. Do we not test against 1.19 in CI?